staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -199,7 +199,11 @@ message PortMapping {
 }
 
 enum MountPropagation {
-    // No mount propagation ("private" in Linux terminology).
+    // No mount propagation ("rprivate" in Linux terminology).
+    // An implementation may raise a warning or an error when "rprivate" is not safely applicable.
+    // However, cri-dockerd (Docker) is known to silently choose "rslave" (i.e., HOST_TO_CONTAINER)
```

Comment:
Is the desire in the API comment change to say that its ok for cri providers to warn/error?

I would expect the CRI provider to publish its known issues separate from API definition.   Can we remove the  cri-dockerd notice as a result?

Reply:
agree the cri-dockerd issue is probably better placed in user facing .. provider specific docs... 

s/An implementation/A runtime/

Reply:
Removed cri-dockerd notice